exercises/matthew_bowles/ex02_oo_basics/inc/Circle.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "Shape.h"
+#include <math.h>
```

Comment:
Why do you need `math.h` here. Only include what is needed. (Look out for this in other files)